pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -193,7 +193,7 @@ class Build {
         }
         def arch_os = "${arch}_${buildConfig.TARGET_OS}"
         jobParams.put('ARCH_OS_LIST', arch_os)
-        jobParams.put('LIGHT_WEIGHT_CHECKOUT', true)
+        jobParams.put('LIGHT_WEIGHT_CHECKOUT', false)
```

Comment:
What's the reason to update to 'false'?

Reply:
If we set `LIGHT_WEIGHT_CHECKOUT ` true, it means AQA repo and branch are hardcoded. It will take the variable that we set in the job parameters.